templates/zerver/emails/password_reset.subject

```diff
@@ -1 +1,3 @@
-Zulip password reset
+{% if attempted_realm %} Failed password reset attempt
+{% else %} Somebody requested a new password for your Zulip account
+{% endif %}
```

Comment:
I guess I think (1) is going to be 98% of the volume, and we should probably optimize the subject line for that case.  Why not just "Password reset for your Zulip account" and then clarify whatever we need to about the possibility someone else triggered this in the body?  

I guess the main reason I'm skeptical is that  I've just never seen anyone use "Somebody" in the subject line of a password reset email.